factorio-map-process.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+set -euo pipefail
+## Defaults
+SERVER_BASE_PATH=/srv/factorio/maps
+MAP_TILES_PATH=images
+FACTORIO_MAP_SCRIPT="$(dirname "${BASH_SOURCE[0]}")/factoriomap.py"
+TARFILE=
+REDUCE=0
+
+function usage() {
+    cat <<EOF
+USAGE
+
+factorio-map-process.sh [OPTIONS] tarfile
+
+DESCRIPTION
+
+This script will process a given tarfile into a set of images used in a leaflet map.
+The default behavior will create a directory for the world maps if one does not exist and copy the example html page in as 'index.html'
+The tar file will be parsed for a map name and a date, separated by underscores (_). Files should be of the format {WorldName}_YYYY-MM-DD.tar.
+    
+OPTIONS
+
+    -t | --tiles <images>           Set the name of the directory to store the tile images in.  Default value is
+                                    'images'
+
+    -s | --server <server path>     Set the base path where the http server files will be stored.  Default value is
+                                    '/srv/factorio/maps'
+
+    -r                              Flag to size-reduce the images folder with the rdfind tool
+EOF
+}
+
+function testRequirements() {
+	if ! [ -x "$(command -v jq)" ]; then
+		echo "ERROR: Requirement not satisfied - jq -"
+		exit 1
+	fi
+
+        if [[ REDUCE -eq 1 && !(-x "$(command -v rdfind)") ]]; then
```

Comment:
If you take the above, this would probably need to change to:
```suggestion
        if [[ REDUCE -eq 1 ]] && ! command -v rdfind >/dev/null; then
```

Reply:
I think the `command` check was giving me issues before.  I'd want to test this one before including it.  Feel free to finish the pull request and change it on your own if you're confident in it.